xxhash.h

```diff
@@ -1452,6 +1452,14 @@ XXH_PUBLIC_API XXH32_hash_t XXH32_hashFromCanonical(const XXH32_canonical_t* src
 
 typedef XXH64_hash_t xxh_u64;
 
+#if !defined (__VMS) \
+  && (defined (__cplusplus) \
+  || (defined (__STDC_VERSION__) && (__STDC_VERSION__ >= 199901L) /* C99 */) )
+    typedef int64_t xxh_i64;
```

Comment:
I presume this type is declared solely for the purpose of compatibility with `_mm512_set1_epi64()` and `_mm256_set_epi64x()`.
But the official type (in Intel documentation) is `__int64`, and I presume it's necessarily defined when the intrinsics are defined. 
Consequently, I believe there is no need to create an intermediate type, just cast to `__int64` directly in the function invocation.

Reply:
On macos, clang declare `_mm512_set1_epi64` and `_mm256_set_epi64x` with type `long long`, and does not define a type of `__int64`.